ProjFS.Mac/PrjFSKext/VnodeCache.cpp

```diff
@@ -78,8 +79,7 @@ kern_return_t VnodeCache_Init()
         return KERN_FAILURE;
     }
 
-    s_entriesCapacity = MAX(MinEntriesCapacity, desiredvnodes * 2);
-    s_entriesCapacity = MIN(MaxEntriesCapacity, s_entriesCapacity);
+    s_entriesCapacity = Clamp(static_cast<uint32_t>(desiredvnodes) * 2, MinEntriesCapacity, MaxEntriesCapacity);
```

Comment:
If you want to avoid the cast, I believe you can multiply `desiredvnodes` by `2u`, the result should be "promoted" to an `unsigned int` implicitly.